files/default/mysql-accounting

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 # Output current disk usage for each database
 DATE=$(date +%Y%m%d)
-ACCOUNTING_DIR="/var/lib/accounting/mysql"
+ACCOUNTING_DIR="/var/lib/mysql-files"
```

Comment:
This path should **not** be changed. Why was it changed?

Reply:
@ramereth I set to this directory because secure_file_priv is set to that directory. If this is set, mysql can only write to that directory. Should I disable secure_file_priv, or maybe change the secure_file_priv to "/var/lib/accounting/mysql"?

Reply:
Are you referring to selinux settings? AFAIK this script runs as root so it shouldn't matter.

Reply:
@ramereth Not sure, it would not converge, however. I got my solution [here](https://stackoverflow.com/questions/34102562/mysql-error-1290-hy000-secure-file-priv-option)